src/test/suite/fixtures/recorded/languages/cpp/clearName.yml

```diff
@@ -0,0 +1,23 @@
+languageId: cpp
+command:
+  version: 1
+  spokenForm: clear name
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: name, includeSiblings: false}
+initialState:
+  documentContents: a += "world"
+  selections:
+    - anchor: {line: 0, character: 6}
+      active: {line: 0, character: 6}
+  marks: {}
+finalState:
+  documentContents: " += \"world\""
```

Comment:
Side note: man yaml is so weird. Imagine how painful it would have been to figure out that we need quotes here but not on the `documentContents` above. Cc/ @somanythings

Reply:
Yeah yikes